.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Publish: Publish to NPM'
+
+on:
+  workflow_dispatch:
```

Comment:
Rearranged and added some explanatory comments to the workflow file in the newest version, as per discussion in our sync. For the context of others: maintaining a manual trigger for the sub-workflow is useful in case something goes wrong with one of them and they need to be re-triggered with new endpoints, or there is an edge case where we want to run an npm publish or github release on its own.